bin/enrich/csv.js

```diff
@@ -228,15 +234,26 @@ const writeHeaderCSV = async (header) => {
  * starts the enrichment process for files CSV
  * @param {*} readStream read the stream of the file you want to enrich
  */
-const enrichmentFileCSV = async (outFile, separatorFile, readStream) => {
+const enrichmentFileCSV = async (outFile, separatorFile, readStream, args) => {
+  if (args.attributes) {
+    checkAttributesCSV();
+  }
+  createFetchAttributes();
   const stat = await fs.stat(readStream.path);
   bar.start(stat.size, 0);
 
-  let loadedBefore = 0;
-  let loadedAfter;
-
   out = outFile;
+
+  // empty the file
+  const ifFileExist = await fs.pathExists(out);
+  if (ifFileExist) {
+    fs.unlink(out, (err) => {
```

Comment:
I guess you could just `await` the file deletion and let it throw the error by itself in case of rejection.